Doppler.HtmlEditorApi.Test/DopplerHtmlDocumentTest.cs

```diff
@@ -0,0 +1,49 @@
+using Doppler.HtmlEditorApi.Storage.DapperProvider.Queries;
+using Doppler.HtmlEditorApi.Storage.DapperProvider;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Net;
+using System.Threading.Tasks;
+using Doppler.HtmlEditorApi.Test.Utils;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using System.Text.Json;
+using Xunit.Abstractions;
+using Xunit;
+using System.Linq;
+
+namespace Doppler.HtmlEditorApi;
+
+public class DopplerHtmlDocumentTest
+{
+    private readonly ITestOutputHelper _output;
+
+    public DopplerHtmlDocumentTest(ITestOutputHelper output)
+    {
+        _output = output;
+    }
+
+    [Fact]
+    public void get_distinct_fields_id()
```

Comment:
What do you think about adding a test scenario for when there are no ids?

Reply:
What should happen if you don't have ids?

Reply:
I think that `Assert.Equal(0, result.Count());` would be fine